index.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Concert Hall</title>
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Candal&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <header>
+    <nav id="nav-container">
+      <div class="mobile-menu">
+        <span class="material-icons hamburger">menu</span>
+      </div>
+      <div class="desktop-menu">
+        <div class="head-menu">
+          <a class="desktop-logo" href="index.html">
+            <img class="logo" src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+        <ul class="nav-list-container">
+          <li ><a class="about-li" href="./pages/about.html">About</a></li>
+          <li><a href="#information">Information</a></li>
+          <li><a href="#bands">Bands</a></li>
+          <li><a href="#partners-container">Partners</a></li>
+          <li><a href="">News</a></li>
+          <li class="box-item"><a class="box-li" href="">CC Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section id="home">
+    <h1 class="home-h1">"Hello! New Music World"</h1>
+    <h2 class="home-h2">GREEN PARK <br> OUTDOOR CONCERT <br> 2022</h2>
+    <p class="concert-description">International concert that presents outstanding performances by the world's greatest artists is taking place in October, in Argentina.</p>
+    <p class="concert-date">2022.10.15(SAT)~16()SUN</p>
+    <p class="concert-place">@ Green Park, Salta - Argentina</p>
+  </section>
+
+  <section id="information">
+    <h2 class="information-h2">Information</h2>
+    <ul class="information-list">
```

Comment:
- [ ] Kindly add more padding to each li's content, currently, everything looks clustered, see screenshot below 🤗 


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_20](https://user-images.githubusercontent.com/69638013/163289137-1553d545-5053-49cf-b4cb-1cbc2eccd42e.png)  | ![Screenshot_19](https://user-images.githubusercontent.com/69638013/163289126-a8ae5f51-af59-4157-9fdd-f63e75280204.png)



### OPTIONAL

- [ ] On hover on each li card, kindly add a border outline to the selected card 🤗